packages/selenium-ide/src/background/background.js

```diff
@@ -230,7 +230,10 @@ browser.runtime.onMessageExternal.addListener(
     let payload = message.payload
 
     payload.sender = sender.id
-    if (message.uri === '/privateConnect' || message.uri === '/privateClose') {
+    if (
+      message.uri === '/private/connect' ||
```

Comment:
you can use something like uri start with /private/ in this condition.

Reply:
done

Reply:
Where is the change?